src/main/java/dictionary/ConstantApp.java

```diff
@@ -0,0 +1,17 @@
+package dictionary;
+
+public class ConstantApp {
+
+    private ConstantApp() {
```

Comment:
Change the name of this class to Constants so we can maintain consistency between different technologies.

Reply:
Changed